stl/inc/algorithm

```diff
@@ -10102,9 +10102,16 @@ namespace ranges {
             if constexpr (forward_range<_Rng> && _Prefer_iterator_copies<iterator_t<_Rng>>) {
                 const auto _Found = _RANGES _Minmax_element_unchecked(
                     _STD move(_UFirst), _STD move(_ULast), _Pass_fn(_Pred), _Pass_fn(_Proj));
+                if constexpr (!is_pointer_v<iterator_t<_Rng>>) {
+                    if (_Found.min == _Found.max) {

```

Comment:
This would not be obvious for readers of code without the context. Add a comment maybe?